docs/ref/models/instances.txt

```diff
@@ -410,7 +410,7 @@ Saving objects
 
 To save an object back to the database, call ``save()``:
 
-.. method:: Model.save(force_insert=False, force_update=False, using=DEFAULT_DB_ALIAS, update_fields=None)
+.. method:: Model.save(*, force_insert=False, force_update=False, using=DEFAULT_DB_ALIAS, update_fields=None)
```

Comment:
As mentioned in the ticket, you should also update the two documentation examples in models.txt, since they would otherwise suggest doing something we've deprecated.